compat-data-schema.md

```diff
@@ -1,334 +1,306 @@
-      }
+### Sub-features
 
-#### `"version_removed"`
-Contains a string with the version number the sub-feature
-stopped to be supported. It may be a Boolean value of (`true` or `false`), or the
-`null` value. If `"version_added"` is set to a Boolean or a `string`, `"version_removed"`
-default value is `false`; if it is `null`, the default value of `"version_removed"`
-is `null` too.
+A sub-feature is the basic entity containing browser compatibility information. As
+explained in the previous section, any feature has at least one sub-feature
+called `basic_support`, but it may have many more.
 
-* Removed in version 10 (start in 3.5):
+A sub-feature may have three properties.
+* A mandatory `support` property for __compat information__.
+An object listing the compatibility information for each browser ([see below](#the-support-object)).
 
-      {
-        "version_added": "3.5",
-        "version_removed": "10"
-      }
+* An optional `desc` property for __sub-feature description__.
+A string containing a human-readable description of the sub-feature.
+It is intended to be used as a caption or title and should be kept short.
+The `<code>` and `<a>` HTML elements can be used.
 
-* Not removed (default if `"version_added"` is not `null`):
+* An optional `status` property for __status information__.
+An object containing information about the stability of the sub-feature:
+Is it a functionality that is standard? Is it stable? Has it been deprecated
+and shouldn't be used anymore ([see below](#status-information)).
 
-     {
-       "version_added": "3.5",
-       "version_removed": false
-     }
+### The `support` object
+Each sub-feature has support information. For each browser identifier,
+it contains a [`support_statement`](#the-support_statement-object) object with
+information about versions, prefixes or alternate names, as well as notes.
 
+#### Browser identifiers
 
-#### `"prefix"`
-Contains the prefix to add to the sub-feature name (default to the empty
-string). Note that leading and trailing `-` must be included.
+The currently accepted browser identifiers are:
+* `webview_android`, Webview, the former stock browser on Android,
+* `chrome`, Google Chrome (on desktops),
+* `chrome_android`, Google Chrome (on Android),
+* `edge`, MS Edge (on Windows),
+* `edge_mobile`, MS Edge, the mobile version,
+* `firefox`, Mozilla Firefox (on desktops),
+* `firefox_android`, Firefox for Android, sometimes nicknamed Fennec,
+* `ie_mobile`, Microsoft Internet Explorer, the mobile version,
+* `ie`, Microsoft Internet Explorer (discontinued),
+* `nodejs` Node.js JavaScript runtime built on Chrome's V8 JavaScript engine,
+* `opera`, the Opera browser (desktop), based on Blink since Opera 15,
+* `opera_android`, the Opera browser (Android version),
+* `safari`, Apple Safari, on Mac OS,
+* `safari_ios`, Apple Safari, on iOS.
+
+No browser identifier is mandatory.
+
+#### The `support_statement` object
+This object is the key element for each browser.
```

Comment:
This is a strange sentence. "key element" makes it sound like it has a technical meaning, like a database key. We could say instead "The `support_statement` object describes the support provided by a single browser type for the given subfeature ".